lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -9,14 +9,11 @@
  * 4) This overlay does not render route paths
  * 5) This overlay has a custom popup on vehicle hover
  */
-import TransitVehicleOverlay from '@opentripplanner/transit-vehicle-overlay'
-import coreUtils from '@opentripplanner/core-utils'
 import { Circle, CircledVehicle } from '@opentripplanner/transit-vehicle-overlay/lib/components/markers/ModeCircles'
-import { Tooltip } from 'react-leaflet'
 import { connect } from 'react-redux'
 import { FormattedMessage, FormattedNumber, injectIntl } from 'react-intl'
-
-const { formatDurationWithSeconds } = coreUtils.time
+import { Tooltip } from 'react-leaflet'
+import TransitVehicleOverlay from '@opentripplanner/transit-vehicle-overlay'
```

Comment:
Sort imports.

Reply:
Sorted in 
092b53f aside from React which I think needs to come first